web/gds-user-ui/src/components/BasicDetailsForm/index.tsx

```diff
@@ -127,6 +140,7 @@ const BasicDetailsForm: React.FC<BasicDetailsFormProps> = ({
             <InputFormControl
               controlId="established_on"
               data-testid="established_on"
+              disabledKeyboardNavigation={false}
```

Comment:
I noticed that I wasn't able to type in a date anymore for `Date of Incorporation/Establishment`.  Should this prop have made sure that users would still be able to type in a date or have I misunderstood the purpose?

Reply:
@daniellemaxwell the goal of these changes is to allow users to change manually the date. The user can only do it by selecting the date picker icon. The goal is to prevent the user from putting a date that is beyond the current date.

Reply:
@daniellemaxwell The intention behind these changes is to restrict manual alteration of the date by users. Users will only have the option to make changes through the date picker icon. This approach aims to prevent users from inputting dates that extend beyond the current date.

Reply:
Reading `false` for `disabledKeyboardNavigation` was a little confusing to me as it made me think that using the keyboard would still be an option. In my mind, true would mean that it is disabled while false would mean that it isn't. Thanks for clarifying.